src/GovUk.Education.ExploreEducationStatistics.Content.Api/Services/ReleaseFileService.cs

```diff
@@ -22,24 +26,40 @@ namespace GovUk.Education.ExploreEducationStatistics.Content.Api.Services
 {
     public class ReleaseFileService : IReleaseFileService
     {
+        private static readonly FileType[] DownloadableFileTypes = {
+            FileType.Ancillary,
+            FileType.Data,
+        };
+
+        private static readonly FileType[] ZipFileTypes = {
+            FileType.Ancillary,
+            FileType.Chart,
+            FileType.Data,
+            FileType.DataGuidance,
+            FileType.Image,
```

Comment:
Do we plan to allow the content images to be included? Don't think they are currently in the All Files zip.

Reply:
Ah good point, I think I was being silly and looked in the wrong place in the publisher when I was deciding what files would be downloadable! Will remove the images.

Reply:
Just checked the update, looks good. 🙂